frontend/src/components/Layout/AuthorizedLayout/MobileMenu/MobileMenuContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, BaseSyntheticEvent } from 'react';
+
+import { WINDOW_SIZE } from 'constants/windowWidth';
+import { useGetWindowSizeLabel } from 'hooks';
+import { TMenuMobileContainerProps } from 'components/Layout/types';
+
+import MobileMenu from './MobileMenu';
+
+const MobileMenuContainer: FC<TMenuMobileContainerProps> = ({ isMobileMenuOpen, ...props }) => {
+  const windowSize = useGetWindowSizeLabel();
+
+  const isMobileWindowSize = [WINDOW_SIZE.sm.name, WINDOW_SIZE.xs.name].includes(windowSize);
+  const isShowMenu = isMobileMenuOpen && isMobileWindowSize;
+
+  const handleSpaceHolderClick = (event: BaseSyntheticEvent) => {
+    event.stopPropagation();
+  };
```

Comment:
can we destructure the event at once, as we discussed?

Reply:
done